pkgs/applications/window-managers/hyprwm/hyprland/plugins.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, callPackage
+, pkg-config
+, gcc13Stdenv
+, hyprland
+}:
+let
+  mkHyprlandPlugin =
+    a@{ pluginName, ... }:
+    gcc13Stdenv.mkDerivation (a // {
+      pname = "${pluginName}";
+      nativeBuildInputs = a.nativeBuildInputs or [ ];
+      buildInputs = [ hyprland ]
+        ++ hyprland.buildInputs
+        ++ (a.buildInputs or [ ]);
+      meta = a.meta // {
+        description = a.meta.description ++
+          "\n\nPlugins can be installed via a plugin entry in the Hyprland NixOS or Home Manager options.";
```

```suggestion
      pname = pluginName;

      nativeBuildInputs = args.nativeBuildInputs or [ ];
      buildInputs = [ hyprland ]
        ++ hyprland.buildInputs
        ++ (a.buildInputs or [ ]);

      meta = args.meta // {
        lognDescription = ''
          Plugins can be installed via a plugin entry in the Hyprland NixOS or Home Manager options.
        '';
```

The amendment to description should go to longDescription because it is to long.

---

fixed

---

according to ofborg checks longdescription does not exist